2019/01/2019-01-18-Fortnite.md

```diff
@@ -0,0 +1,81 @@
+**Are you the copyright owner or authorized to act on the copyright owner’s behalf?**
+
+I, [private], am authorized to act on behalf of the owner of an exclusive right that is being infringed.
+
+**Please provide a detailed description of the original copyrighted work that has allegedly been infringed. If possible, include a URL to where it is posted online.**
+
+Epic Games, Inc. (“Epic”) has created and licenses the proprietary video game software known as Fortnite (http://www.epicgames.com/fortnite). In addition to Epic’s copyright in Fortnite, Epic has registered the Fortnite mark (registration numbers: 4481629 and 5375076, among others), and has various common law trademark rights related to Fortnite (collectively, “Intellectual Property Rights”).
+
+
+
+**What files should be taken down? Please provide URLs for each file, or if the entire repository, the repository’s URL:**
+
+https://github.com/mq1n/FORTNITE-HACK-1
+
+https://github.com/wouback/FORTNITE-HACK
+
+https://github.com/Roshly/Skin-Royale
+
+
+
+**Have you searched for any forks of the allegedly infringing files or repositories? Each fork is a distinct repository and must be identified separately if you believe it is infringing and wish to have it taken down.**
+
+
+
+**Is the work licensed under an open source license? If so, which open source license? Are the allegedly infringing files being used under the open source license, or are they in violation of the license?**
+
+
+
+Epic Games, Inc. is the sole owner of the copyright to “Fortnite” in accordance with Title 17 of the U.S. Code. Thus, Epic Games possesses the exclusive right to reproduce, modify, post, and/or distribute those materials or to grant others permission to exercise one or more of those rights. Epic Games highly values our intellectual property, because it is an extremely important component of our business and overall success. The software at issue enables unauthorized access to copyrighted material, and thus constitutes contributory and/or induced copyright infringement.
```

Comment:
>The software at issue enables unauthorized access to copyrighted material, and thus constitutes contributory and/or induced copyright infringement.

This should be fun in court.

Filing a DMCA requires you certify that you are the owner of the content to be taken down. It can not be used to take down facilitation software as the filer is not the copyright owner of the software.